.ci/install_clear_image.sh

```diff
@@ -16,34 +16,31 @@
 
 set -e
 
-if [ "$#" -ne 2 ]; then
-    echo "Usage: $0 CLEAR_RELEASE PATH"
-    echo "       Install the clear rootfs image from clear CLEAR_RELEASE in PATH."
-    exit 1
-fi
-
-clear_release="$1"
-install_path="$2"
-image=clear-${clear_release}-containers.img
-cc_img_link_name="clear-containers.img"
-base_url="https://download.clearlinux.org/releases/${clear_release}/clear"
-
-echo "Download clear containers image"
-curl -LO "${base_url}/${image}.xz"
-
-echo "Validate clear containers image checksum"
-curl -LO "${base_url}/${image}.xz-SHA512SUMS"
-sha512sum -c ${image}.xz-SHA512SUMS
-
-echo "Extract clear containers image"
-unxz ${image}.xz
-
-sudo mkdir -p ${install_path}
-echo "Install clear containers image"
-sudo install -D --owner root --group root --mode 0755 ${image} ${install_path}/${image}
-
-echo -e "Create symbolic link ${install_path}/${cc_img_link_name}"
-sudo ln -fs ${install_path}/${image} ${install_path}/${cc_img_link_name}
-
-# clean up
-rm -f ${image} ${image}.xz-SHA512SUMS
+repo_owner="clearcontainers"
+repo_name="osbuilder"
+cc_image_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
+
+function download_image() {
```

Comment:
I see that this function has a lot in common with this one:
https://github.com/clearcontainers/tests/blob/master/.ci/install_clearcontainers_kernel.sh#L34

Do you think it would be possible to have a common function called `download_component()` (or something similar) and add it to the `.ci/lib.sh` and then just call it here?

Reply:
Yep, agree with that .